src/register/tests/RegistrationPage.test.jsx

```diff
@@ -191,8 +191,10 @@ describe('RegistrationPage', () => {
       expect(registrationPage.find('div[feedback-for="country"]').text()).toEqual(emptyFieldValidation.country);
 
       const alertBanner = 'We couldn\'t create your account.Please check your responses and try again.';
+      registrationPage.find('RegistrationPage').setState({ failureCount: 1 });
```

Comment:
why are we setting the `failureCount` explicitly? Shouldn't this be set automatically when we click the submit button?

Reply:
It should set automatically but not setting up. That's why we need to set `failureCount` explicitly to cover this case in unit test .